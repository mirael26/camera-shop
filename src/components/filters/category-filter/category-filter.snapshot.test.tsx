import { renderWithReduxAndRouter } from '../../../test/helpers/render-with-redux-and-router';
import CategoryFilter from './category-filter';

test('CagetoryFilter component renders correctly', () => {
  const {asFragment} = renderWithReduxAndRouter(<CategoryFilter/>);
  expect(asFragment()).toMatchSnapshot();
});
