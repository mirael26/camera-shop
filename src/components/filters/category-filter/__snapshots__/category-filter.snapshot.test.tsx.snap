// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CagetoryFilter component renders correctly 1`] = `
<DocumentFragment>
  <fieldset
    class="catalog-filter__block"
  >
    <legend
      class="title title--h5"
    >
      Категория
    </legend>
    <div
      class="custom-checkbox catalog-filter__item"
    >
      <label>
        <input
          name="Фотоаппарат"
          type="checkbox"
        />
        <span
          class="custom-checkbox__icon"
        />
        <span
          class="custom-checkbox__label"
        >
          Фотокамера
        </span>
      </label>
    </div>
    <div
      class="custom-checkbox catalog-filter__item"
    >
      <label>
        <input
          name="Видеокамера"
          type="checkbox"
        />
        <span
          class="custom-checkbox__icon"
        />
        <span
          class="custom-checkbox__label"
        >
          Видеокамера
        </span>
      </label>
    </div>
  </fieldset>
</DocumentFragment>
`;
